src/zabapgit_object_clas.prog.abap

```diff
@@ -196,6 +206,11 @@ CLASS lcl_object_oriented_base IMPLEMENTATION.
     ENDIF.
   ENDMETHOD.
 
+  METHOD lif_object_oriented_object~get_includes.
+    "Subclass responsibility
+    RETURN.
```

Comment:
in cases like this I usually do `ASSERT 0 = 1.` to force the developer to overload the method, any thoughts?